src/views/verify.tsx

```diff
@@ -12,44 +89,49 @@ export const Verify: React.FunctionComponent = () => {
   }, [rawDocument]);
 
   const [fragments, setFragments] = useState<VerificationFragment[]>([]);
-  const [network, setNetwork] = useState<Network>("ropsten");
+  const [network, setNetwork] = useState<network>("ropsten");
```

Comment:
Am I right that `ropsten` will be the default network? Should we make the default network as Goerli?🤔

We have to change anyway in 3 months time. Just thinking if we should start with this.😛